### PR TITLE
Request maintainer role

### PR DESCRIPTION
**Pull request by Waschndolos**
# Description

[https://github.com/jenkinsci/prometheus-plugin/](https://github.com/jenkinsci/prometheus-plugin/)
@markyjackson-taulia
@waschndolos

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [x] [All newly added users have logged in to Artifactory and Jira at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Review by markyjackson-taulia — CHANGES_REQUESTED** [review]
You many line feeds, need to remove one. I will approve once that is done @Waschndolos

**Comment by markyjackson-taulia** [comment]
Once the line feeds are corrected, I approve this change to step down as the maintainer of this plugin. Thank you @Waschndolos for stepping up.
I also apologize to the users for neglecting this plugin as of late.

**Comment by Waschndolos** [comment]
@markyjackson-taulia No problem. I see you have a lot of other things in your pipeline. Happy to help. I fixed the linefeed

**Pull request merged**
